version-control/introduction/tutorial.md

```diff
@@ -38,21 +38,23 @@ Wikipedia also holds a history of all changes.
 
 ## Why do you need Version Control?
 
-- When used on a regular basis, version control helps you to store your files safely. It makes it easy to figure out what broke your code as you can roll back to a previous version and work out when things last worked as expected.
+- When used on a regular basis, version control helps you to store your files safely. It makes it easy to figure out what broke your code, as you can roll back to a previous version and work out when things last worked as expected.
+    - With no version control in place you'll only have one copy of your file, then when it breaks there's no way to get back to good code!
```

Comment:
rather than the phrase "good code" could we say "working code" instead please/

Reply:
Great - will change that phrase.